fmriprep/data/reports-spec-anat.yml

```
@@ -0,0 +1,47 @@
+package: fmriprep
+sections:
+- name: Summary
+  reportlets:
+  - bids: {datatype: figures, desc: summary, suffix: T1w}
+- name: Anatomical
+  reportlets:
+  - bids:
+      datatype: figures
+      desc: conform
+      extension: [.html]
+      suffix: T1w
+  - bids: {datatype: figures, suffix: dseg}
+    caption: This panel shows the final, preprocessed T1-weighted image,
+      with contours delineating the detected brain mask and brain tissue segmentations.
+    subtitle: Brain mask and brain tissue segmentation of the T1w
+  - bids: {datatype: figures, space: .*, suffix: T1w, regex_search: True}
+    caption: Spatial normalization of the T1w image to the <code>{space}</code> template.
+    description: Results of nonlinear alignment of the T1w reference one or more template
+      space(s). Hover on the panels with the mouse pointer to transition between both
+      spaces.
+    static: false
+    subtitle: Spatial normalization of the anatomical T1w reference
+  - bids: {datatype: figures, desc: reconall, suffix: T1w}
+    caption: Surfaces (white and pial) reconstructed with FreeSurfer (<code>recon-all</code>)
+      overlaid on the participant's T1w template.
+    subtitle: Surface reconstruction
+- name: About
+  reportlets:
+  - bids: {datatype: figures, desc: about, suffix: T1w}
+  - custom: boilerplate
+    path: '{reportlets_dir}/logs'
```

I don't think the logs are written out there for fMRIPrep. Also, I don't think the CITATION.html file is written out into logs:

```suggestion
    path: '{output_dir}'
```

---

Yes, the CITATION.html is written inside logs. I had the debug that line to make it work, so I'm sure about this.